### ISSUE TITLE
Zipfiles in Asset finder

### ISSUE THREAD

**Issue by galensoft** [Dovyski/Codebot#93]
When looking in the asset finder, there are several zips. 

If I'm not mistaken, it is not possible to do much with a zip in Codebot right now.
A suggestion is to either not show zips or unzip them when adding them to the project

**Comment by Dovyski** [comment]
I am sorry about that. As you pointed out, you can't do anything with zip files at the moment. I was planning to add an option like "Unzip file here" when you right-click any zips in the files panel. Useful for other use cases as well. What do you think?

Alternatively, I can unzip asset bundles on the server and then add all files when the asset is added to the project.

**Comment by eriknielsen** [comment]
Yeah, a "Unzip file here" sounds good. Also adding the size of a file in the Asset Finder would be useful.

Great work by the way!

**Comment by Dovyski** [comment]
Awesome, I will go with those two tasks then: unzip option and file size in the Asset Finder.

> Great work by the way!

Thank you very much! 😄
